google-cloud-spanner-executor/clirr-ignored-differences.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- see http://www.mojohaus.org/clirr-maven-plugin/examples/ignored-differences.html -->
+<differences>
+  <difference>
+    <differenceType>8001</differenceType>
+    <className>com/google/cloud/executor/spanner/CloudClientExecutor</className>
```

Comment:
why are we ignoring these differences?

Reply:
This is because of change in the package name from earlier.
`com/google/cloud/executor/spanner` to `com/google/cloud/spanner/executor/`

This was done to maintain consistency with other class name in this repo
For ex: `com.google.cloud.spanner.*`

Reply:
Usually we don't exclude breaking changes. We only exclude things which `clirr` identified by mistake but we know its not breaking anything. In this case, `clirr` actually identified a breaking change.

Reply:
As discussed, we will not be modifying the package names to avoid breaking changes and ignore consistency.